package.json

```diff
@@ -19,7 +19,8 @@
     "react-native-reanimated": "^1.7.0",
     "react-native-safe-area-context": "^0.7.3",
     "react-native-screens": "^2.2.0",
-    "react-native-vector-icons": "^6.6.0",
+    "react-native-splash-screen": "^3.2.0",
+    "react-native-vector-icons": "^6.6.0W",
```

Comment:
There is no such version on NPM ---> 6.6.0W 
Why you change it ?

Reply:
Missclick 😓